src/instructlab/model/evaluate.py

```diff
@@ -0,0 +1,74 @@
+# Standard
+import json
+import os
+
+# Third Party
+import click
+
+# First Party
+from instructlab import utils
+
+
+@click.command()
+@click.option(
+    "--data-dir",
+    help="Base directory where data is stored.",
+    default="./taxonomy_data",
+    show_default=True,
+)
+@click.option(
+    "--model-dir",
+    help="Base directory where model is stored.",
+    default="ibm-merlinite-7b-mlx-q",
+    show_default=True,
+)
+@click.option(
+    "--adapter-file",
+    help="LoRA adapter to use for test. Set to 'None' to force only testing behavior from before training.",
+    default="auto",
+    show_default=True,
+)
+@utils.macos_requirement(echo_func=click.secho, exit_exception=click.exceptions.Exit)
+# pylint: disable=function-redefined
+def evaluate(data_dir, model_dir, adapter_file):
+    """Runs basic test to ensure model correctness"""
+    # pylint: disable=C0415
+    # Local
+    from ..train.lora_mlx.lora import load_and_train
+
+    if adapter_file == "auto":
+        adapter_file = os.path.join(model_dir, "adapters.npz")
+    elif adapter_file.lower() == "none":
+        adapter_file = None
+
+    adapter_file_exists = adapter_file and os.path.exists(adapter_file)
+    if adapter_file and not adapter_file_exists:
+        print(
```

Comment:
Won't block on this, but why are we using print statements here instead of the logger?

Reply:
This was the existing code, so let's address it in a future PR